cypress/support/cypress-commands/commands.js

```diff
@@ -1263,3 +1263,96 @@ Cypress.Commands.add('methodOrEventResponseValidation', (validationType, request
     }
   });
 });
+
+/**
+ * @module commands
+ * @function validateMethodOrEventResponseForV2
+ * @description Validating the event or method response
+ * @param {String} validationType - Determines event or method validation
+ * @param {String} method - API name
+ * @param {*} validationJsonPath - Contains an array or string with the path of the response value that needs to be validated.
+ * @param {String} contentObject - Contains source of truth for content validation.
+ * @param {String} appId - app identified name.
+ * @param {String} errorContent - Holds the error content validation object when error is expected.
+ * @param {String} eventExpected - Determines whether expecting for a event or not.
+ * @example
```

Comment:
validateMethodOrEventResponseForV2 this can be removed and changed to validateMethodOrEventResponseForJs

Reply:
addressed the comment